### DISCUSSION TITLE
Need guide/fix for configuring OPENAI_KEY in self-hosted supabase

### DISCUSSION THREAD

**Discussion by Bananamilk452** [supabase/supabase#22560] [Feature Requests]
i know there is almost same discussions #21621, but this is about self-hosted docker. not the cli.

I was trying to use supabase ai to build SQL queries, but adding OPENAI_KEY in several config files doesn't working.

1. Add OPENAI_KEY in docker/.env - doesn't work
2. Add OPENAI_KEY in environment of supabase-studio in docker/docker-compose.yml - doesn't work

Only solution was adding OPENAI_KEY in apps/studio/.env file which is in working supabase-studio container.

I think, OPENAI_KEY should be working by adding in the docker/.env file.

**Comment by noname-oni** [comment]
Same issue :(
